serverlessworkflow/modules/ROOT/pages/integrations/expose-metrics-to-prometheus.adoc

```diff
@@ -1 +1,117 @@
-//= Exposing the workflow base metrics to Prometheus
\ No newline at end of file
+Exposing the workflow base metrics to Prometheus
+================================================
+11-03-2022
+:compat-mode!:
+// Metadata:
+:description: Exposing the workflow base metrics to Prometheus
+:keywords: kogito, workflow, quarkus, metrics, prometheus
+// links
+:openshift_micrometer_url: https://quarkus.io/blog/micrometer-prometheus-openshift
+:dashbuilder_url: https://www.dashbuilder.org/
+:grafana_url: https://grafana.com/
+:quarkus_micrometer_url: https://quarkus.io/guides/micrometer
+:openshift_monitoring_url: https://docs.openshift.com/container-platform/4.7/monitoring/enabling-monitoring-for-user-defined-projects.html
+
+Kogito Serverless Workflow generate metrics that can be consumed by Prometheus and visualized by dashboard tools such as link:{openshift_micrometer_url}[Openshift], link:{dashbuilder_url}[Dashbuilder] and link:{grafana_url}[Grafana].
+
+This document describes how to enable and expose metrics to Prometheus.
+
+== Enabling metrics
+
+For adding metrics to your Serverless Workflow application just add the dependency `org.kie.kogito:kogito-addons-quarkus-monitoring-prometheus` and the metrics will be available in `/q/metrics` URL.
+
+.Dependency to be added to `pom.xml` to enable metrics
+[source,xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-monitoring-prometheus</artifactId>
+</dependency>
+----
+
+== Consuming metrics from Openshift
```

Comment:
Can we have the Kubernetes/Minikube counterpart as well? I imagine it's just a matter of installing the operator there.

Reply:
@jesuino, are you planning to add these sections?